lib/pbench.egg-info/entry_points.txt

```diff
@@ -0,0 +1,11 @@
+[console_scripts]
```

Comment:
I will just remove it

Reply:
The `.gitignore` file _should_ have prevented this file from being added to your commit (at least, by default).  However, the entry in `.gitignore` is
```
*.egg-info
```
Since it's actually a directory, it should have a trailing slash -- i.e.,
```
*.egg-info/
```
Riya, I recommend that you make this change to `.gitignore` (and include it in this PR or in a separate one) and see if it helps.

Reply:
yeah it did prevent the file from being added. I will create a separate PR with this change.